### PR TITLE
CB-12078 Revert - CB Scaling API that works with internal crns.

### PR DESCRIPTION
**Pull request by foldik**
See detailed description in the commit message.

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java [hunk 1]
```diff
@@ -119,27 +120,31 @@ public AutoscaleStackV4Response getInternalAutoscaleClusterByName(String name, @
     }
 
     @Override
-    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
+    @DisableCheckPermissions
```
Comment (lnardai) [inline core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java:123]: I think these can be removed:
    @PreAuthorize("hasRole('AUTOSCALE')")

    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)

Comment (foldik) [reply core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java:123]: It was the original version before the change, I wouldn't risk it to introduce a new failure.

File: core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackApiViewService.java [hunk 2]
```diff
@@ -123,6 +124,7 @@ public StackApiView save(StackApiView stackApiView) {
         ));
     }
 
+    @PreAuthorize("hasRole('INTERNAL')")
```
Comment (horadla23) [inline core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackApiViewService.java:127]: this isn't needed

**Review by horadla23 — APPROVED** [review]
LGTM

**Comment by foldik** [comment]
Tested with schedule based scaling. Worked fine, the original API level exception disappeared.

**Pull request merged**